test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+import boto3
+import pytest
+
+from test.test_utils import get_repository_and_tag_from_image_uri, LOGGER
+
+
+def test_dlc_major_version_label(image, region):
+    ecr_client = boto3.client("ecr", region_name=region)
+
+    image_repository, image_tag = get_repository_and_tag_from_image_uri(image)
+    response = ecr_client.batch_get_image(
+        repositoryName=image_repository,
+        imageIds=[{"imageTag": image_tag}],
+        acceptedMediaTypes=["application/vnd.docker.distribution.manifest.v1+json"],
```

Comment:
can you provide more info on why this is needed?

Reply:
Adding comment.

Basically this is a format in which the response needs to be sent, so that the image LABELS can be found on the manifest.